src/Coordination/Changelog.cpp

```diff
@@ -1027,37 +1078,27 @@ void Changelog::writeThread()
             assert(current_writer);
 
             batch_append_ok = current_writer->appendRecord(buildRecord(append_log->index, append_log->log_entry));
+            ++pending_appends;
         }
         else
         {
             const auto & flush = std::get<Flush>(write_operation);
```

Comment:
Add check/assert that it's really `Flush` event?

Reply:
`std::get` already throws if we try to get invalid type, but I can still add check/assert